mpc-core/src/proto/core/ot.rs

```diff
@@ -149,16 +167,28 @@ impl From<ot::ExtReceiverSetup> for ExtReceiverSetup {
         Self {
             ncols: s.ncols as u32,
             table: s.table,
+            x: s.x.to_vec(),
+            t0: s.t0.to_vec(),
+            t1: s.t1.to_vec(),
         }
     }
 }
 
 impl From<ExtReceiverSetup> for ot::ExtReceiverSetup {
```

Comment:
I agree. Just to make sure, @sinui0 , the section of the code you highlighted is not what needs to be changed, because the direction there is core model -> prost model, right?

Reply:
`impl From<ExtReceiverSetup> for ot::ExtReceiverSetup` I believe this is prost -> core.

I'm _definitely_ going to refactor this after this PR to make it less confusing.